.github/workflows/test.yaml

```
@@ -31,7 +31,7 @@ jobs:
       - name: Setup micromamba
         uses: mamba-org/setup-micromamba@v1
         with:
-          environment-file: ${{ runner.os == 'Windows' && 'environment_cpu_win.yaml' || 'environment.yaml' }}
+          environment-file: ${{ runner.os == 'Windows' && 'environment_win.yaml' || 'environment.yaml' }}
```

```suggestion
          environment-file: 'environment.yaml'
```

you might as well simplify your CI file altogether.

---

and your instructions!